styles/_uswds-theme-custom-styles.scss

```diff
@@ -323,17 +323,35 @@ HOME
 }
 
 // .gov
-.usa-search--domain span {
-  display: flex;
-  background-color: #AFD5E6;
-  padding-left: 1rem;
-  padding-right: 1rem;
-  border: 1px solid #565c65;
-  border-left: none;
-  align-items: center;
-  color: color('primary-darker');
-  font-weight: bold;
-  height: 2.125rem;
+.usa-search--domain {
+  span {
+    display: flex;
+    padding-left: 1rem;
+    padding-right: 1rem;
+    border: 1px solid #565c65;
+    
+    align-items: center;
+    color: color('primary-darker');
+    font-weight: bold;
+    height: 2.125rem;
+  }
+}
+
+.usa-search--domain-primary {
+  @extend .usa-search--domain;
```

Comment:
found @extend useful in creating CSS "variants" if anyone wants to make variants of elements in the future